packages/src/lib/block-link/block-link-title-description/block-link-title-description.component.scss

```
@@ -31,10 +32,6 @@ sdg-anchor ::ng-deep a {
       font-size: 16px;
       visibility: hidden;
       margin-left: 8px;
```

Les deux stratégies sont bonne mais parfois j'ai une préférence pour conserver le media query local, c'est plus lisibile.
```suggestion
      margin-left: 8px;
 
      @media only screen and (max-width: breakpoints.max-width(mobile)) {
        visibility: visible;
      }
```